app/azure/Dashboard/DashboardItem.tsx

```diff
@@ -0,0 +1,23 @@
+import { Widget, WidgetBody } from '@/portainer/components/widget';
```

Comment:
should be in `app/portainer/components/DashboardItem.tsx`

Reply:
i wondered about that, now the item will be able to be reused in other dashboards when they are migrated 👍🏻